src/main/java/org/intellij/sdk/codesync/ui/dialogs/PricingAlertDialog.java

```diff
@@ -25,18 +29,22 @@ public class PricingAlertDialog extends DialogWrapper {
     public PricingAlertDialog(Boolean isOrgRepo, Boolean canAvailTrial, String pricingURL, Project project){
         super(project, true);
         this.pricingURL = pricingURL;
-
         if (canAvailTrial) {
-            upgradeButtonText = "Try Pro for free";
-            secondaryMessage = isOrgRepo ? Notification.TRY_ORG_PRO_FOR_FREE: Notification.TRY_PRO_FOR_FREE;
+            upgradeButtonText = isOrgRepo ? NotificationButton.TRY_TEAM_FOR_FREE: NotificationButton.TRY_PRO_FOR_FREE;
+            secondaryMessage = isOrgRepo ? Notification.TRY_TEAM_PLAN_FOR_FREE: Notification.TRY_PRO_FOR_FREE;
         } else {
-            upgradeButtonText = "Upgrade to Pro";
+            upgradeButtonText = NotificationButton.UPGRADE_PLAN;
             secondaryMessage = isOrgRepo ? Notification.UPGRADE_ORG_PRICING_PLAN: Notification.UPGRADE_PRICING_PLAN;
         }
 
         setTitle(this.title);
     }
 
+    public PricingAlertDialog(Boolean canAvailTrial){
+        this(true, false, CODESYNC_PRICING_URL, null);
```

Comment:
Where are we using canAvailTrial? 🤔 Seems like we are passing hard coded False
Also, why are we passing true for isOrgRepo?

Reply:
Oh yes, I was passing different values for our cases. Forgot to revert. Updating

Reply:
I see, we should be testing with actual code and create data on the server we need for local testing so that actual code is tested on different cases